person_capture/gui_app.py

```
@@ -2017,188 +2020,201 @@ def _compose_dataset_crop(
+            preferred = {
+                "close": ["1:1", "2:3"],
+                "upper": ["2:3", "1:1"],
+                "body": ["2:3", "1:1", "3:2"],
+                "base": ["1:1", "2:3"],
+            }.get(profile, ["1:1", "2:3"])
+            out: list[str] = []
+            for rs in preferred + [str(r).strip() for r in (ratio_candidates or []) if str(r).strip()]:
+                try:
+                    parse_ratio(rs)
+                except Exception:
+                    continue
```

Ruff (0.15.11)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)